ui/Screen/DesignSystemGuide.svelte

```diff
@@ -1074,6 +1075,10 @@
       <Swatch>
         <Stats branches={2} commits={12} contributors={4} />
       </Swatch>
+
+      <Swatch>
+        <Urn urn="5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu" />
+      </Swatch>
```

Comment:
Could also add the other variants here.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/731/commits/4a9b3c5116431a32169b623ededfdef1be279cc0